src/globals.ts

```diff
@@ -95,10 +101,15 @@ export const ExecutionTags = (() => {
 
   const addTag = (key, value, shouldLogErrors = true) => {
     try {
-      logger.debug(`Adding tag: ${key} - ${value}`);
-      if (!validateTag(key, value, shouldLogErrors)) return false;
-      // @ts-ignore
-      global.tags.push({ key: normalizeTag(key), value: normalizeTag(value) });
+      if (isLambdaTraced()) {
+        logger.debug(`Adding tag: ${key} - ${value}`);
+        if (!validateTag(key, value, shouldLogErrors)) return false;
+        // @ts-ignore
```

Comment:
why do we need this `@ts-ignore`? can we put the reason inline?

Reply:
Good question, it was in the original code, I don't know what was the reason